services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> list[CinemaHall]:
+    return CinemaHall.objects.all()
```

Comment:
are u sure about list type annotation ? 
it seems to me, that method all returns queryset)

Reply:
ffs, I knew I'm going to forget to change this :(